plugin-server/src/worker/ingestion/event-pipeline/cookielessServerHashStep.ts

```diff
@@ -214,6 +253,8 @@ export async function cookielessServerHashStep(
 
         event.properties['$session_id'] = sessionState.sessionId
 
+        stripPIIProperties(event)
```

Comment:
We're always unconditionally stripping these off, but there might be reasons to use cookieless BUT have permission to store PII, right? Is this something you intend to implement eventually, or are we ok always stripping these off?